doc/source/user_guide/groupby.rst

```diff
@@ -418,6 +418,12 @@ You can also include the grouping columns if you want to operate on them.
 
 .. _groupby.iterating-label:
 
+.. note::
```

Comment:
Can you put this before `See :ref:`timeseries.iterating-label`.` in the section?

Reply:
> `See :ref:`timeseries.iterating-label`.`

This is in the "Iterating through groups" section. Should I move it to that section?

Reply:
Oh I see, sorry. Could you put this before ``.. _groupby.iterating-label:`` then?